### ISSUE TITLE
Add CDF curves to report

### ISSUE THREAD

**Issue by rihorn2** [fairlearn/fairlearn#236]
For regression models, add view with CDF curves and minimum distance between sub-group metric

**Comment by romanlutz** [comment]
Plotting cumulative distribution functions by group could be useful for regression scenarios. We've actually done this in a [notebook](https://github.com/fairlearn/fairlearn/blob/main/notebooks/Mitigating%20Disparities%20in%20Ranking%20from%20Binary%20Data.ipynb) (search for `plot_and_compare_cdfs`).

This [post](https://stackoverflow.com/questions/25577352/plotting-cdf-of-a-pandas-series-in-python) has some interesting responses that may be simpler since it would just involve using the prediction `pandas.Series` and its plotting functionality.

I imagine a function for this could live in the `metrics` module (or `experimental` if people want to be careful). Input parameters would include `y` and `sensitive_features`.

**Comment by romanlutz** [comment]
Certainly! I would strongly suggest trying to prototype this based on the notebook first and to share a draft PR based on that before going into documentation so that people can comment on the API and see whether this matches their mental model. Might save you some time, too. 
I'll assign it to you @Thomasq99 .
